frontend/fingertips-frontend/README.md

```diff
@@ -119,3 +127,35 @@ These server pages are responsible for making any calls to fetch any data. Then
 The page components are pure react components and must have the `use client` directive at the top. This is needed for the purpose of using the `govuk-react` component library. This library uses `styled-components` and react hooks and therefore need to be client components.
 
 However, Next.js will still use these component to render the page server-side.
+
+## Auto generate scripts
+
+CAUTION: Running these scripts will overwrite previously generated code.
+
+### Generating the API Client
+
+The following script will autogenerate the api client code from an openapi spec.
+
+```bash
+npm run generate:api-client"
+```
+
+The generated code is held within `generated-sources/api-client` folder. Please do not make any manual changes to code within this folder. This is auto-generated and will be overwritten if the script is ran again.
+
+`openapitools.json` contains configuration that the script uses. This includes where to find the open-api spec as input and where the output is generated. You can change the output path if you wish to see the generated code and not overwrite the previous version.
+
+### Generating mock handlers
+
+The following script will autogenerate mock handlers from an open-api spec.
+
+```bash
+npm run generate:mocks
```

Comment:
Yeah which is why I'm not too sure about this one just yet. Since it does require some manual intervention and can't be used out the box. Just seems useful to autogenerate the handlers for all the endpoints. Which we could do manually without much effort. I'm more inclined to take this one out.

Reply:
I suspect we'll probably end up just manually adding entries for new endpoints, rather than completely regenerating the handlers, so I'm OK if you prefer to take it out. I'm also OK with leaving it in, to give us the option. I'll leave it up to you!